MiniApp/Classes/core/Downloader/MiniAppDownloader.swift

```diff
@@ -29,9 +29,7 @@ class MiniAppDownloader {
     }
 
     fileprivate func cleanApp(_ appId: String, for version: String) {
-        if self.miniAppClient.environment.isPreviewMode == false {
-            self.miniAppStorage.cleanVersions(for: appId, differentFrom: version, status: self.miniAppStatus)
-        }
+        miniAppStorage.cleanVersions(for: appId, differentFrom: version, status: miniAppStatus)
```

Comment:
So we are going to download the Mini app every time irrespective of the Preview or Published mode?

Reply:
no, the check is done line 37 @rleojoseph ;)